Userland/Libraries/LibArchive/Tar.h

```diff
@@ -74,12 +74,10 @@ static StringView get_field_as_string_view(char const (&field)[N])
 template<size_t N, class TSource>
 static void set_field(char (&field)[N], TSource&& source)
 {
-    if constexpr (requires { source.characters_without_null_termination(); }) {
-        memcpy(field, source.characters_without_null_termination(), min(N, source.length()));
-    } else {
-        auto success = source.copy_characters_to_buffer(field, N);
-        VERIFY(success);
-    }
+    VERIFY(N >= source.length());
+    memcpy(field, StringView(source).characters_without_null_termination(), min(N, source.length()));
```

Comment:
We already verified that `N` is greater to or equal to `source.length()`. This means that `min(N, source.length()) == source.length()`.